qdrant-landing/content/blog/qdrant-1.14.x.md

```diff
@@ -0,0 +1,232 @@
+
+In **e-commerce** you may want to boost products from a specific manufacturer—perhaps because you have a promotion or need to clear inventory. With this update, you can easily influence ranking using metadata like `brand` or `stock_status`.
+
+> The **Score-Boosting Reranker** allows you to combine vector-based similarity with **business or domain-specific logic** by applying a **rescoring step** on top of the standard semantic or distance-based ranking.
+
+As you structure the query, you can define a `formula` that references both existing scores (like cosine similarities) and additional payload data (e.g., timestamps, location info, numeric attributes). Let's take a look at some examples:
+
+### Idea 1: Prioritizing Website Content
+
+Let's say you are trying to improve the search feature for a documentation site, such as our [**Developer Portal**](https://qdrant.tech/documentation/). You would chunk and vectorize all your documentation and store it in a Qdrant collection.
+
+**Figure 1:** Any time someone types in **hybrid queries**, you want to show them the most relevant result at the top.
+![reranking](/blog/qdrant-1.14.x/website-search.png)
+
+**Reranking** can help you prioritize the best results based on user intent.
+
+Your website collection can have vectors for **titles**, **paragraphs**, and **code snippet** sections of your documentation. You can create a `tag` payload field that indicates whether a point is a title, paragraph, or snippet. Then, to give more weight to titles and paragraphs, you might do something like:
+
+```
+score = score + (is_title * 0.5) + (is_paragraph * 0.25)
+```
+
+**Above is just sample logic - but here is the actual Qdrant API request:**
+
+```bash
+POST /collections/{collection_name}/points/query
+{
+    "prefetch": {
+        "query": [0.2, 0.8, ...],   // <-- dense vector for the query
+        "limit": 50
+    },
+    "query": {
+        "formula": {
+            "sum": [
+                "$score",
+                {
+                    "mult": [
+                        0.5,
+                        { "key": "tag", "match": { "any": ["h1","h2","h3","h4"] } }
+                    ]
+                },
+                {
+                    "mult": [
+                        0.25,
+                        { "key": "tag", "match": { "any": ["p","li"] } }
+                    ]
+                }
+            ]
+        }
+    }
+}
+```
+
+### Idea 2: Reranking Most Recent Results
+
+One of the most important advancements is the ability to prioritize recency. In many scenarios, such as in news or job listings, users want to see the most recent results first. Until now, this wasn’t possible without additional work: *you had to fetch all the data and manually filter for the latest entries on their side*.
```

Comment:
```suggestion
One of the most important advancements is the ability to prioritize recency. In many scenarios, such as in news or job listings, users want to see the most recent results first. Until now, this wasn’t possible without additional work: *you had to fetch all the data and manually filter for the latest entries on your side*.
```

I might be wrong, but it would sound more naturally to me.